smoketest/scripts/cli/test_service_ssh.py

```diff
@@ -213,5 +213,50 @@ def ssh_send_cmd(command, username, password, host='localhost'):
         usernames = [x[0] for x in getpwall()]
         self.assertNotIn(test_user, usernames)
 
+    def test_ssh_dynamic_protection(self):
+        """
+        check sshguard service
+        """
+        SSHGUARD_CONFIG = '/etc/sshguard/sshguard.conf'
+        SSHGUARD_PROCESS = 'sshguard'
+        block_time = '123'
+        detect_time = '1804'
+        port = '22'
+        threshold = '10'
+
+        self.cli_set(base_path + ['dynamic-protection', 'block-time', block_time])
+        self.cli_set(base_path + ['dynamic-protection', 'detect-time', detect_time])
+        self.cli_set(base_path + ['dynamic-protection', 'threshold', threshold])
+
+        # commit changes
+        self.cli_commit()
+
+        # Check configured port
+        tmp = get_config_value('Port')
+        self.assertIn(port, tmp)
+
+        # Check sshgurad service
+        self.assertTrue(process_named_running(SSHGUARD_PROCESS))
+
+        sshguard_lines = [
+            'THRESHOLD=10',
+            'BLOCK_TIME=123',
+            'DETECTION_TIME=1804'
+        ]
+
+        tmp_sshguard_conf = read_file(SSHGUARD_CONFIG)
+
+        for line in sshguard_lines:
+            self.assertIn(line, tmp_sshguard_conf)
+
+        # Delete service ssh dynamic-protection
+        # but not service ssh itself
+        self.cli_delete(base_path + ['dynamic-protection'])
+        self.cli_commit()
+
+        self.assertFalse(process_named_running(SSHGUARD_PROCESS))
+        self.assertTrue(process_named_running(PROCESS_NAME))
```

Comment:
This line is already tested in `tearDown()` which is invoked after this function exists.

Reply:
Done